src/graph/helper.cc

```diff
@@ -988,7 +988,7 @@ GraphPtr_t graphBuilder(const ProblemSolverPtr_t& ps,
     assert(robot.robotFrames(od.name).size() != 0);
     for (const FrameIndex& f : robot.robotFrames(od.name)) {
       if (model.frames[f].type != ::pinocchio::JOINT) continue;
-      const JointIndex j = model.frames[f].parent;
+      const JointIndex j = model.frames[f].parentJoint;
```

Comment:
The change looks reasonable but I'm surprised. This must have been used successfully in the past right ?

Reply:
It is related to the pinocchio upgrade to version 3. `parent` has become `parentJoint`. The old naming is deprecated.